src/webviews/webview-side/interactive-common/variableExplorer.tsx

```diff
@@ -348,6 +348,14 @@ export class VariableExplorer extends React.Component<IVariableExplorerProps, IV
                 } else if (variable.count) {
                     newSize = variable.count.toString();
                 }
+                let value = variable.value;
+                if (variable.type === 'str' && variable.value) {
+                    if (variable.value[0] === ' ') {
```

Comment:
I would rather we found where this space was added and removed that there instead of patching this here.

Else tomorrow if we remove this leading space from the other end (which evere code/layer adds that), then we'd end up stripping the first empty character (here) from strings which would be wrong.
Unless someone remembers to remove this patchy code, which I think is less likely to happen.

Reply:
Done here: https://github.com/microsoft/vscode-jupyter/pull/10732/commits/767759d3f2ad6c05d8bd75264315c4e1ba8083b5